src/gui/clips/MidiClipView.cpp

```diff
@@ -46,6 +46,7 @@
 namespace lmms::gui
 {
 
+static int const s_beatStepButtonOffset = 4;
```

Comment:
Could be `constexpr` (and we tend to use UpperCamelCase for constants, with the `s_` prefix denoting mutable static variables).

Reply:
Fixed with commit ffb2a669.